build-tools/scripts/MSBuildReferences.projitems

```diff
@@ -1,13 +1,13 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <!--Import this file in projects needing to reference Microsoft.Build.*.dll -->
+  <!-- Import this file in projects needing to reference Microsoft.Build.*.dll -->
   <ItemGroup>
     <!-- Compile against Microsoft.Build* NuGet refs, but do not copy to OutputDir. -->
     <PackageReference Include="Microsoft.Build" Version="15.9.20" IncludeAssets="compile" />
     <PackageReference Include="Microsoft.Build.Framework" Version="15.9.20" IncludeAssets="compile" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.9.20" IncludeAssets="compile" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.9.20" IncludeAssets="compile"/>
-    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
-    <PackageReference Include="Xamarin.Build.AsyncTask" Version="0.3.4" />
+    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" GeneratePathProperty="true" />
+    <PackageReference Include="Xamarin.Build.AsyncTask" Version="0.3.4" GeneratePathProperty="true" />
```

Comment:
What does `%(PackageReference.GeneratePathProperty)` do?

Reply:
This sets an msbuild property to the package path in the `nuget.g.props` file that is imported by the project. It is used later in  https://github.com/xamarin/xamarin-android/pull/3928/files#diff-e388143943a5a1fdab55ecb6f09169e7R340 to copy "unexpected" NuGet package content to the `OutputDirectory` so that these files can be picked up by our installers.